yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -165,6 +165,14 @@ func packAndSendAssessMigrationPayload(status string, errMsg string) {
 				ObjectCount: len(feature.Objects),
 			}
 		})),
+		UnsupportedPlPgSqlObjects: callhome.MarshalledJsonString(lo.Map(assessmentReport.UnsupportedPlPgSqlObjects, func(plpgsql UnsupportedFeature, _ int) callhome.UnsupportedFeature {
+			groupedObjects := groupByObjectName(plpgsql.Objects)
+			return callhome.UnsupportedFeature{
+				FeatureName:      plpgsql.FeatureName,
+				ObjectCount:      len(lo.Keys(groupedObjects)),
+				TotalOccurrences: len(plpgsql.Objects),
```

Comment:
How about just `TotalCount` ? in-line with `ObjectCount` ?

Reply:
I think `TotalCount` does not indicate the actual meaning of the count, So I thought `TotalOccurences` was better, as it indicated the occurrences of this feature in the schema.

Reply:
Yeah, I was also a little iffy about `TotalCount` . 👍